hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/mapred-default.xml

```diff
@@ -1266,6 +1266,29 @@
   </description>
 </property>
 
+<property>
+  <name>mapreduce.shuffle.pathcache.max-weight</name>
+  <value>10 * 1024 * 1024</value>
```

Comment:
Thanks @jiwq for the reply and the update.

One comment: the style cannot be parsed by org.apache.hadoop.conf.Configuration.
```suggestion
  <value>10485760</value>
```

+1 if that is addressed.
Sorry I forgot to review for a long time.

Reply:
Done.